bigtable/client/row_reader_test.cc

```diff
@@ -316,13 +319,39 @@ TEST_F(RowReaderTest, FailedStreamWithNoRetryThrows) {
   bigtable::RowReader reader(
       client_, "", bigtable::RowSet(), bigtable::RowReader::NO_ROWS_LIMIT,
       bigtable::Filter::PassAllFilter(), std::move(retry_policy_),
-      std::move(backoff_policy_), metadata_update_policy_,
+      std::move(backoff_policy_), std::move(metadata_update_policy_),
       std::move(parser_factory_));
 
   EXPECT_THROW(reader.begin(), std::exception);
 }
 #endif  // GOOGLE_CLOUD_CPP_HAVE_EXCEPTIONS
 
+TEST_F(RowReaderTest, FailedStreamWithNoRetryThrowsNoExcept) {
```

Comment:
I am missing a test to verify that an exception is raised if the error is not retrieved, but it could be because this is a large file.  Can you point it to me if it does exist already and add it if it does not?